Libplanet.Tests/Net/Protocols/TestSwarm.cs

```diff
@@ -53,6 +53,8 @@ public class TestSwarm : ISwarm
             _random = new Random();
         }
 
+        public IEnumerable<BoundPeer> Peers => Protocol.Peers;
```

Comment:
Should this be necessarily `public`? If so it's better to have a single sentence docs as well.

Reply:
I made it public since `TestSwarm` is a test class which in `Libplanet.Tests`.